### PR TITLE
Add unit tests for without_attr function

### PR DESCRIPTION
**Pull request by praneen**
# Description 

Addition of tests for without_attr() object utility.

- [x] Tests for a flat level dictionary.
- [x] Tests for a nested dictionary.
- [ ] Add exceptions for incompatible data types.
- [ ] Tests for exceptions.
- [ ] Tests for empty cases.
- [ ] Tests for iterables.

### CODE REVIEW

File: tests/util/test_object.py [hunk 1]
```diff
@@ -233,3 +233,40 @@ def test_merge_v2():
...
+    dict1 = {
+        "Name": "Geeks",
+        "Gender": "Male",
+        "Age": "55",
+        "Address": {
+            "Street": "Charkhal",
+            "District": "Kathmandu",
+        }
+    }
+    dict1 = {
```
Comment (sanjevShakya) [inline tests/util/test_object.py]: this should be dict2 right?

Comment (praneen) [reply tests/util/test_object.py]: Yes, thanks.

Comment (praneen) [reply tests/util/test_object.py]: @sanjevShakya It passed due to variable override. :D

**Comment by kabirbaidhya** [comment]
@praneen You have a formatting issue to fix. Check here :point_right: https://travis-ci.com/leapfrogtechnology/datamegh/builds/149438927#L443-L506.

Run `make format` which should fix those and you can push them.

**Comment by praneen** [comment]
> @praneen You have a formatting issue to fix. Check here  https://travis-ci.com/leapfrogtechnology/datamegh/builds/149438927#L443-L506.
> 
> Run `make format` which should fix those and you can push them.

Thanks will do. @kabirbaidhya will the precommit hooks address this later on? Is this something @Basanta78 is also working on?

**Comment by kabirbaidhya** [comment]
@praneen Precommit hook is removed. It isn't required since we we have CI which stops you. All you need to do is `make format` and make sure you do `make test` before you push.

**Comment by praneen** [comment]
> @praneen Precommit hook is removed. It isn't required since we we have CI which stops you. All you need to do is `make format` and make sure you do `make test` before you push.

`make format` doesn't seem to be working. But let me see what I can do.

**Comment by praneen** [comment]
> > @praneen Precommit hook is removed. It isn't required since we we have CI which stops you. All you need to do is `make format` and make sure you do `make test` before you push.
> 
> `make format` doesn't seem to be working. But let me see what I can do.

Nevermind I was oustide the virtual env.

**Comment by praneen** [comment]
@kabirbaidhya Sure will do.

**Pull request merged**